### DISCUSSION TITLE
Disable removing extra whitespaces

### DISCUSSION THREAD

**Discussion by Mahonzu** [SubtitleEdit/subtitleedit#8412] [Q&A]
Hello! I use this program to make animated lyrics for songs, and sometimes it requires a lot of whitespaces in a row. My problem is that when I hit save, all these whitespaces get deleted. Is there a way to disable this? I have searched the entire settings menu and haven't found anything. 
Thanks!

**Comment by GrampaWildWilly** [comment]
![#01](https://github.com/SubtitleEdit/subtitleedit/assets/75588964/ab13545a-bea1-45b9-9483-e1cb89caab38)

Or maybe there's some other place in SE that the function occurs.  I'm not an expert.

**Reply by Mahonzu** [reply]
Thanks for your reply! Unfortunately, this doesn't seem to be a problem since I have to manually run "Fix common errors" for it to do anything. Thanks for your time though!

**Comment by niksedk** [comment]
> Hello! I use this program to make animated lyrics for songs, and sometimes it requires a lot of whitespaces in a row. My problem is that when I hit save, all these whitespaces get deleted. Is there a way to disable this? I have searched the entire settings menu and haven't found anything. Thanks!

What subtitle format are you using? You loose the white space by... loading/saving/?

**Reply by Mahonzu** [reply]
Thanks for your reply! I use the .lrc format and can confirm that extra whitespaces disappear when I save (even if I haven't made any changes to it).

I've attached a file that which on save will get cleared of extra spaces
[example.txt](https://github.com/SubtitleEdit/subtitleedit/files/15435765/example.txt)
(it's in .txt because github won't let me upload .lrc)
1. Open a file in any other text editor to confirm that extra spaces are there
2. Open file in SubtitleEdit and save it (Ctrl+S)
3. Extra whitespaces are gone
